Tests/UK/Test.cpp

```diff
@@ -111,862 +111,864 @@ int main()
-        exam.push_back(q133);
 
+	exam.push_back(q9);
+	exam.push_back(q10);
+	exam.push_back(q11);
+	exam.push_back(q12);
+	exam.push_back(q13);
+	exam.push_back(q14);
+	exam.push_back(q15);
+	exam.push_back(q16);
+	exam.push_back(q17);
+	exam.push_back(q18);
+	exam.push_back(q19);
+	exam.push_back(q20);
+	exam.push_back(q21);
+	exam.push_back(q22);
+	exam.push_back(q23);
+	exam.push_back(q24);
+	exam.push_back(q25);
+	exam.push_back(q26);
+	exam.push_back(q27);
+	exam.push_back(q28);
+	exam.push_back(q29);
+	exam.push_back(q30);
+	exam.push_back(q31);
+	exam.push_back(q32);
+	exam.push_back(q33);
+	exam.push_back(q34);
+	exam.push_back(q35);
+	exam.push_back(q36);
+	exam.push_back(q37);
+	exam.push_back(q38);
+	exam.push_back(q39);
+	exam.push_back(q40);
+	exam.push_back(q41);
+	exam.push_back(q42);
+	exam.push_back(q43);
+	exam.push_back(q44);
+	exam.push_back(q45);
+	exam.push_back(q46);
+	exam.push_back(q47);
+	exam.push_back(q48);
+	exam.push_back(q49);
+	exam.push_back(q50);
+	exam.push_back(q51);
+	exam.push_back(q52);
+	exam.push_back(q53);
+	exam.push_back(q54);
+	exam.push_back(q55);
+	exam.push_back(q56);
+	exam.push_back(q57);
+	exam.push_back(q58);
+	exam.push_back(q59);
+	exam.push_back(q60);
+	exam.push_back(q61);
+	exam.push_back(q62);
+	exam.push_back(q63);
+	exam.push_back(q64);
+	exam.push_back(q65);
+	exam.push_back(q66);
+	exam.push_back(q67);
+	exam.push_back(q68);
+	exam.push_back(q69);
+	exam.push_back(q70);
+	exam.push_back(q71);
+	exam.push_back(q72);
+	exam.push_back(q73);
+	exam.push_back(q74);
+	exam.push_back(q75);
+	exam.push_back(q76);
+	exam.push_back(q77);
+	exam.push_back(q78);
+	exam.push_back(q79);
+	exam.push_back(q80);
+	exam.push_back(q81);
+	exam.push_back(q82);
+	exam.push_back(q83);
+	exam.push_back(q84);
+	exam.push_back(q85);
+	exam.push_back(q86);
+	exam.push_back(q87);
+	exam.push_back(q88);
+	exam.push_back(q89);
+	exam.push_back(q90);
+	exam.push_back(q91);
+	exam.push_back(q92);
+	exam.push_back(q93);
+	exam.push_back(q94);
+	exam.push_back(q95);
+	exam.push_back(q96);
+	exam.push_back(q97);
+	exam.push_back(q98);
+	exam.push_back(q99);
+	exam.push_back(q100);
+	exam.push_back(q101);
+	exam.push_back(q102);
+	exam.push_back(q103);
+	exam.push_back(q104);
+	exam.push_back(q105);
+	exam.push_back(q106);
+	exam.push_back(q107);
+	exam.push_back(q108);
+	exam.push_back(q109);
+	exam.push_back(q110);
+	exam.push_back(q111);
+	exam.push_back(q112);
+	exam.push_back(q113);
+	exam.push_back(q114);
+	exam.push_back(q115);
+	exam.push_back(q116);
+	exam.push_back(q117);
+	exam.push_back(q118);
+	exam.push_back(q119);
+	exam.push_back(q120);
+	exam.push_back(q121);
+	exam.push_back(q122);
+	exam.push_back(q123);
+	exam.push_back(q124);
+	exam.push_back(q125);
+	exam.push_back(q126);
+	exam.push_back(q127);
+	exam.push_back(q128);
+	exam.push_back(q129);
+	exam.push_back(q130);
+	exam.push_back(q131);
+	exam.push_back(q132);
+	exam.push_back(q133);
+
+
+	system("chcp 65001  > nul");
```

Comment:
why so?

Reply:
To fix the problem with encoding, when you use Code blocks. You can delete it.